app/services/album.js

```diff
@@ -0,0 +1,19 @@
+'use strict';
+
+const axios = require('axios'),
+  config = require('../../config'),
+  errors = require('../errors'),
+  logger = require('../logger'),
+  url = config.common.albumsApi.url;
+
+exports.getAll = async source => {
+  const albumData = await axios.get(`${url}${source}`).catch(err => {
+    logger.error(err);
+    if (err.response.status === 300)
```

Comment:
It's ok for some cases, remember that status codes are a range of values like 3XX, 4XX and 5XX